examples/kafkatest_verifiable_client.cpp

```diff
@@ -457,6 +457,17 @@ void msg_consume(RdKafka::KafkaConsumer *consumer,
               " [" << (int)msg->partition() << "]  at offset " <<
               msg->offset() << std::endl;
 
+        RdKafka::Headers *headers = msg->get_headers();
```

Comment:
There is no need for headers support in the verifiable_client, please remove these changes.

kafkatest_verifiable_client is used to run the Java system tests with the librdkafka client.